cypress/integration/regression/main-table-regression.spec.ts

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable no-undef */
+import { CommonSelectors } from '../../../src/components/data-testid/common-selectors.cy'
+import { datasetPage } from '../../page-objects/app/datasets-page'
+import { mainTablePage } from '../../page-objects/app/main-table-page'
+import { variantDrawerPage } from '../../page-objects/app/variant-drawer-page'
+import { Timeouts } from '../../page-objects/lib/timeouts.cy'
+
+describe('Regression test of the main table | step 1', () => {
+  const link = '/ws?ds=PGP3140_wgs_panel_hl'
```

Comment:
You use "PGP3140_wgs_panel_hl" 9 times in this file. It's a hint what to change.